clap_generate/src/generators/mod.rs

```diff
@@ -0,0 +1,323 @@
+mod shells;
+
+// Std
+use std::io::Write;
+
+// Internal
+use clap::*;
```

Comment:
Import only what we require not the entire set

Reply:
Will fix in a new PR.